### PR TITLE
Move to npm 7 & update global configs

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,14 +46,12 @@
 		"@nextcloud/logger": "^2.0.0",
 		"@nextcloud/router": "^2.0.0",
 		"calendar-js": "git+https://github.com/nextcloud/calendar-js.git",
```
Comment (artonge) [inline package.json:48]: This is making CI fail

```
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/nextcloud/calendar-js.git
npm ERR! 
npm ERR! Warning: Permanently added the RSA host key for IP address '140.82.113.3' to the list of known hosts.
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
```

Maybe move to HTTP ?

Comment (artonge) [reply package.json:48]: https://github.com/npm/cli/issues/2610

Comment (artonge) [reply package.json:48]: Hacky wrokaround: https://github.com/npm/cli/issues/2610#issuecomment-851347583

Comment (raimund-schluessler) [reply package.json:48]: Another option would be `"calendar-js": "github:nextcloud/calendar-js",` I think.

Comment (szaimen) [reply package.json:48]: Or use the npm package? See https://github.com/nextcloud/calendar/pull/3371

Comment (skjnldsv) [reply package.json]: Done! :)

**Comment by skjnldsv** [comment]
Ah damn, the styleguide doesn't use npm7, I can adjust

**Comment by skjnldsv** [comment]
Good to go!

**Review by marcoambrosini — APPROVED** [review]

**Review by artonge — APPROVED** [review]

**Pull request merged**